build/commands/lib/config.js

```diff
@@ -338,6 +338,11 @@ Config.prototype.buildArgs = function () {
 
   if (process.platform === 'darwin') {
     args.allow_runtime_configurable_key_storage = true
+    // ICF currently (cr98) causes a barrage of linker warning and significantly
+    // slows down the build. Tracked in https://crbug.com/1279639
+    if (!this.isOfficialBuild()) {
+      args.use_icf = false
```

Comment:
what about pulling the upstream patch instead which seems to be solving the issue? https://chromium.googlesource.com/chromium/src/+/9dc9fa84434a1d13450a8612a199fa26ce0a364a%5E%21/#F0

Reply:
upstream patch is definitely ok for stuff like this

Reply:
Reverted this in 8654d13 and added the upstream patch in 87a765a